index-de.html

```diff
@@ -195,7 +200,7 @@ <h3 id="commit">Commit</h3>
   <p>Ein vierter Fall ist, ein Commit mit <a href="#detached">detached HEAD</a>,
   wird weiter unten erklärt.</p>
 
-  <h3 id="checkout">Checkout</h3>
+  <h3 id="checkout">Checkout (Ausprobieren)</h3>
```

Comment:
@HybridDog Would you mind having a look here? It sometimes is not so easy to find precise translations that make sense... They are not used in explanations anyway but I agree that understanding the command names is valuable.